spring-boot/src/main/java/org/springframework/boot/logging/LoggingSystem.java

```diff
@@ -143,6 +143,15 @@ public LoggerConfiguration getLoggerConfiguration(String loggerName) {
 	}
 
 	/**
+	 * Delete configuration for a {@link LoggingSystem}'s logger.
+	 * @param loggerName the name of the logger
+	 * @since 1.5.0
+	 */
+	public void deleteLoggerConfiguration(String loggerName) {
+		throw new UnsupportedOperationException("Unable to delete logger configuration");
```

Comment:
The actual implementation of deleting a logger seems to be missing. The various `LoggingSystem` subclasses need to be updated to override this method with a logging system-specific implementation.